channeld/channel_wire.csv

```diff
@@ -180,3 +180,7 @@ channel_dev_memleak_reply,,leak,bool
 channel_fail_fallen_behind,1028
 channel_fail_fallen_behind,,remote_per_commitment_point,struct pubkey
 
+# Handle a channel specific feerate base ppm configuration
+channel_json_feerates,1029
+channel_json_feerates,,feerate_base,u32
+channel_json_feerates,,feerate_ppm,u32
```

Comment:
The word 'json' is anomolous here.  Sure, the feerate change is probably driven by a JSON command somewhere, but that's not anything to do with channeld; 'channel_set_feerates' would be a better name?

Reply:
`channel_set_feerates` is ambiguous, as there already exists a `channe_feerates (1027)` message which is meant to set the onchain rates. This was my reason for calling it `channel_json_feerates`. 

So make it *a bit* clearer im going to call it now: `channel_specific_feerates`